src/zope/app/form/browser/metaconfigure.py

```diff
@@ -113,7 +114,7 @@ def _normalize(self):
         if self.template is not None:
             self.template = os.path.abspath(str(self.template))
             if not os.path.isfile(self.template):
-                raise ConfigurationError("No such file", self.template)
+                raise ZopeXMLConfigurationError("No such file", self.template)
```

Comment:
I don't understand why this was changed. This is not coming from XML (necessarily). Semantically, it's just an ordinary `ConfigurationError`. Can you offer details please?

Reply:
Good catch, I must have replaced by accident during editing. Unfortunately, this branch is not covered by tests, so nothing failed. I will restore the old behaviour.